### DISCUSSION TITLE
How do I unsubscribe from a service?

### DISCUSSION THREAD

**Discussion by somemone0** [ovh/manager#5133] [Q&A] [answered]
I have a VPS that it turned out i did not need. How can I unsubscribe from it?

**Comment by antleblanc** [comment, marked as answer]
Hi @TheCoder11!

From the Dashboard, you can click on "Products and services" in the "Shortcuts"
section (from right panel). It will redirect you to the "My services" page.

Then you can use filters to list only "Virtual private servers".

You'll have access to an "Actions" column. 

Hope it helps you,
Thank you!
